### PR TITLE
QACI-139 Add Payara Samples to PR tests (public Jenkins)

### PR DESCRIPTION
**Pull request by AlanRoth**
# Description
New Feature for public Jenkins

Added new Payara Samples stage to public jenkins, PRs will be tested against Payara Samples to catch any test failures at the PR stage. 

# Testing

### New tests
Payara Samples added to PR testing.

### Testing Performed
Ran Jenkins script against PR #4584 - No specific reason why this PR

### Testing Environment
Public Jenkins

### CODE REVIEW

**Comment by AlanRoth** [comment]
jenkins test please

**Comment by AlanRoth** [comment]
jenkins test please

File: Jenkinsfile [hunk 1]
```diff
@@ -67,6 +67,25 @@ pipeline {
...
+            steps {
+                echo '*#*#*#*#*#*#*#*#*#*#*#*#  Running test  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
+                sh """mvn -V -B -ff clean install -Ppayara-server-managed \
+                -Dpayara.version=${pom.version} \
+                -Dglassfish.home=\"${pwd()}/appserver/distributions/payara/target/stage/payara5/glassfish\" \
+                -Dpayara_domain=${DOMAIN_NAME} \
+                -Djavax.net.ssl.trustStore=${env.JAVA_HOME}/jre/lib/security/cacerts \
+                -Djavax.xml.accessExternalSchema=all \
+                -Dsurefire.rerunFailingTestsCount=2 \
+                -f appserver/tests/payara-samples """
```
Comment (MarkWareham) [inline Jenkinsfile:80]: May also need to build any required artifacts

File: Jenkinsfile [hunk 2]
```diff
@@ -67,6 +67,25 @@ pipeline {
                 }
             }
         }
+        stage('Run Payara Samples Tests') {
+            steps {
+                echo '*#*#*#*#*#*#*#*#*#*#*#*#  Running test  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
+                sh """mvn -V -B -ff clean install -Ppayara-server-managed \
```
Comment (MarkWareham) [inline Jenkinsfile:73]: I would expect 'verify' to be as far as we need to go in Maven lifecycle

**Pull request merged**